test/package.json

```diff
@@ -4,36 +4,44 @@
   "private": true,
   "scripts": {
     "test": "redis-cli flushall && node --preserve-symlinks test test",
+    "debug": "redis-cli flushall && node --preserve-symlinks test test --log-level debug",
+    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm test",
+    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm test",
+    "coverage-report": "nyc report",
     "watch": "grunt watch"
   },
   "dependencies": {
     "async": "^2.4.1",
-    "bedrock": "^1.12.0",
+    "bedrock": "^3.1.1",
     "bedrock-docs": "^3.0.0",
-    "bedrock-express": "^2.0.6",
-    "bedrock-identity": "^4.6.0",
+    "bedrock-express": "^3.2.0",
+    "bedrock-https-agent": "^1.1.0",
     "bedrock-injector": "^1.0.0",
     "bedrock-jobs": "^3.0.0",
-    "bedrock-key": "^4.0.0",
-    "bedrock-ledger-agent": "^2.0.0",
-    "bedrock-ledger-consensus-continuity": "^3.0.0",
+    "bedrock-jsonld-document-loader": "^1.0.1",
+    "bedrock-key": "^5.1.4",
```

Comment:
Is `bedrock-key` actually required here?  I think that goes away when we eliminate `bedrock-identity`.

Reply:
Removed it here and good catch: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity-storage/pull/6/commits/4ed786192522b59c60fa5bdb1c3c5f6e1fb0f6c9